test/functional/suite_test.go

```diff
@@ -243,4 +245,35 @@ var _ = BeforeEach(func() {
 	DeferCleanup(th.DeleteNamespace, namespace)
 	//Let's create the osp-secret in advance (in common to all the test cases)
 	DeferCleanup(k8sClient.Delete, ctx, CreateGlanceSecret(glanceName.Namespace, SecretName))
+
+	// Create MariaDBAccount and Secret.
```

Comment:
so I got functional tests to pass (:tada: ) but I had to do some dark magic to get there.

Obviously I will add a "CreateMariaDBAccount" utility function to mariadb/api/test/crd.  just calling that here might be good enough for now, but the base controller here does create this account object as well.

Reply:
This is a great news, I understand it's hard to get a full e2e deployment with dependencies across the operators, so I think functional tests represent a great start.
Thanks for your effort and I'll review it in a bit.